### PR TITLE
Create Subscription.re

### PR DESCRIPTION
**Pull request by hew**

### CODE REVIEW

File: src/Subscription.re [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  [@bs.module] external gql: ReasonApolloTypes.gql = "graphql-tag";
+
+  type error = {. "message": string};
+
+  [@bs.deriving abstract]
+  type options = {
+    [@bs.optional]
+    skip: bool,
+    [@bs.optional]
+    onSubscriptionData: unit => unit,
```
Comment (fakenickels) [inline src/Subscription.re]: nice, maybe we can bind already `subscriptionData` and `client` https://github.com/trojanowski/react-apollo-hooks/blob/master/src/useSubscription.ts#L18?

Comment (hew) [reply src/Subscription.re]: Can you explain further what you mean?

Comment (fakenickels) [reply src/Subscription.re]: it's because onSubscriptionData receives an object with two keys `subscriptionData` and `client`

Comment (fakenickels) [reply src/Subscription.re]: I think it's ok for now, I'll merge like this

**Comment by fakenickels** [comment]
@all-contributors add @hew for code

**Pull request merged**